api/v1/authentication_types.go

```diff
@@ -100,8 +100,30 @@ type AuthenticationOIDC struct {
 	// +optional
 	// +kubebuilder:validation:Enum=Verify;InsecureSkip
 	EmailVerification *EmailVerificationType `json:"emailVerification,omitempty"`
+
+	// PromptTypes is an optional list of string values that specifies whether the identity provider prompts the end user
+	// for re-authentication and consent. See the RFC for more information on prompt types:
+	// https://openid.net/specs/openid-connect-core-1_0.html. The defined values are:
+	// ["None", "Login", "Consent", "SelectAccount"].
+	// +optional
+	PromptTypes []PromptType `json:"promptTypes,omitempty"`
 }
 
+// PromptType is a value that specifies whether the identity provider prompts the end user for re-authentication and
+// consent. See the RFC for more information on prompt types: https://openid.net/specs/openid-connect-core-1_0.html
+type PromptType string
```

Comment:
Looking at what is rendered in the API Reference, we should list the allowed values in the comment on this type.

Reply:
I still don't see the allowed values in this comment.

Reply:
Added. I used the "One of: ..." way for showing the options. This seems to have been used a lot in the recent past.

Reply:
Ok, just for my info, does the api reference do anything special with that?